core/classifiers/classifier_utils.py

```diff
@@ -60,15 +59,40 @@ def extract_svm_parameters(clf):
     }
 
 
-def unicode_validator_for_classifier_data(var):
+def unicode_validator_for_classifier_data(classifier_data):
     """Validates that incoming object contains unicode literal strings."""
```

Comment:
Add args docstring explaining the format of classifier_data (or informing where to find its format specification)

Reply:
There is no specific format. It depends on each classifier.